src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  .star {
+    background-image: url(../images/star-active.svg);
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: auto;
+  }
+
+  .star:last-child {
```

Comment:
Consider using styles from the previous task. Modifiers should be working

Reply:
fix it plz because now it looks like 
![image](https://github.com/mate-academy/layout_catalog/assets/96265844/d0521d84-ac63-4075-8485-45810f7fc150)